core/src/main/scala/chisel3/Module.scala

```diff
@@ -399,7 +394,9 @@ package experimental {
     final def toAbsoluteTarget: IsModule = {
       _parent match {
         case Some(parent) => parent.toAbsoluteTarget.instOf(this.instanceName, toTarget.module)
-        case None => toTarget
+        case None =>
+          // FIXME Special handling for Views - evidence of weirdness of .toAbsoluteTarget
```

Comment:
Maybe more of a comment for why this is necessary?

Reply:
Will push a longer comment.